utils/core-modules/contracts/interfaces/IAssociatedSystemsModule.sol

```diff
@@ -1,9 +1,42 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-/// @title Allows for the registration and tracking of auxillery contracts which also follow the proxy architecture
+/**
+ * @title Module for connecting a system with other associated systems.
+ *
+ * Associated systems can me managed or unmanaged.
+ *
+ * Managed systems are connected via a proxy, which means that their implementation can be updated, and the system controls the execution context of the associated system. Example, an snxUSD token connected to the system, and controlled by the system.
```

Comment:
It specifically has to be this kind of proxy, right?

Reply:
The system creates the proxy for you, and it is UUPS yes.
Unmanaged systems can be behind whatever, a proxy of any type, no proxy, etc